### ISSUE TITLE
relias training 

### ISSUE THREAD

**Issue by jamesrad4732** [Clutch152/scripts#16]
dont know how to message but i want to learn how to do what you are doing and do it in relias training. thanks

**Comment by iEatNoodlez** [comment]
For RELIAS ACADEMY?

**Comment by jamesrad4732** [comment]
yes. at least specifically for terraceview.training.reliaslearning.com thanks!Sent from my Samsung Galaxy smartphone.
-------- Original message --------From: iEatNoodlez_ <notifications@github.com> Date: 6/4/19  2:43 PM  (GMT-08:00) To: Clutch152/scripts <scripts@noreply.github.com> Cc: jamesrad4732 <paul.baluyot4@gmail.com>, Author <author@noreply.github.com> Subject: Re: [Clutch152/scripts] relias training  (#16) For RELIAS ACADEMY?

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/Clutch152/scripts/issues/16?email_source=notifications\u0026email_token=AF5CPTD7GR235DTQDCMK2FDPY3OX5A5CNFSM4HMWKQSKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODW56XSQ#issuecomment-498854858",
"url": "https://github.com/Clutch152/scripts/issues/16?email_source=notifications\u0026email_token=AF5CPTD7GR235DTQDCMK2FDPY3OX5A5CNFSM4HMWKQSKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODW56XSQ#issuecomment-498854858",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by jamesrad4732** [comment]
i am willing to give access codes so terraceview.training.reliaslearning.com be studied. thanks

**Comment by ExoticMidnight** [comment]
What’s the username and password?

**Comment by jamesrad4732** [comment]
baluyot3767
pwd is 3767

**Comment by jamesrad4732** [comment]
baluyot3767
pwd 3767

**Comment by jamesrad4732** [comment]
wanted to follow up if anybody was able to help with an answer. tia.

**Comment by ExoticMidnight** [comment]
I have not but I will ASAP

**Comment by jamesrad4732** [comment]
thank you!

**Comment by qkhas4** [comment]
to help get relias done faster:

start course.
right click the course in the popup and click 'inspect'.
click through the topics.
if the 'next arrow' is disabled
         - you can either wait 5 seconds.
         - type: "next_page.disabled=false" in the console (located on the bottom when you clicked 'inspect' earlier.
it doesn't matter if you click the right answers in any of the topics except for the final exam.
when you take the exam go back to the console
type correctAnswers
Press enter and it will give you a number
count that many numbers from top to bottom and click the answer
click next.

**Comment by qkhas4** [comment]
goto console and paste this:

for(let i=1; i<20; i++)
{
next_page.disabled=false;
next_page.click();
setTimeout(1000);
}
